sources/packages/backend/apps/api/src/route-controllers/student/student.institutions.controller.ts

```diff
@@ -68,4 +70,26 @@ export class StudentInstitutionsController extends BaseController {
   ): Promise<StudentProfileAPIOutDTO> {
     return this.studentControllerService.getStudentProfile(studentId);
   }
+
+  /**
+   * This controller returns all student documents uploaded by student uploader.
+   * @param studentId student id.
+   * @returns list of student documents.
+   */
+  @Get(":studentId/documents")
+  async getInstitutionStudentFiles(
+    @Param("studentId", ParseIntPipe) studentId: number,
+  ): Promise<InstitutionStudentFileAPIOutDTO[]> {
```

Comment:
I would like to have the studentId to be validated by the API before proceeding any further like this in the begining of the method. 

```ts
const student = await this.studentService.studentExists(studentId);
    if (!student) {
      throw new NotFoundException("Student not found.");
    }
```

Reply:
@dheepak-aot is it not supposed to be already ensured when the authorization decorator is in place?

Reply:
@sh16011993 ignore my previous comment. As @andrewsignori-aot mentioned, it will be handled by the decorator.